dir.c

```diff
@@ -658,6 +658,38 @@ void clear_exclude_list(struct exclude_list *el)
 	memset(el, 0, sizeof(*el));
 }
 
+static void trim_trailing_comments(char *buf)
+{
+	char *p, *last_octothorpe = NULL;
```

Comment:
_octothorpe_? This can be hard to understand to native speaker. Isn't better just `hash`, or `hash_mark`?

```suggestion
	char *p, *last_hash = NULL;
```